solara/components/slider.py

```diff
@@ -21,7 +22,8 @@ def SliderInt(
     max: int = 10,
     step: int = 1,
     on_value: Optional[Callable[[int], None]] = None,
-    thumb_label=True,
+    thumb_label: bool | Literal['always'] | None = 'always',
```

Comment:
I'm not 100% sure, but I think this means you cannot do:
```python
x: str = "always"
...
SliderInt(thumb_label=always)
```

But I understand what you want, I just don't know if this is the way to do it in Python

Reply:
FYI: Literal is in typing_extensions

Reply:
Thanks, haven't used that extension actually.

The code you linked should work if you set input to "x", if I'm right.

Regarding "show_end_points" how about giving multiple option and defaulting to showing end-points only?

I think min&max makes a lot of sense for numeral ranges. It makes the UI clearer and easier to work with. A screenshot of the app will also tell the full story.
That's my opinion, but I'll do what you prefer as you're the maintainer 😊👍